files/es/web/accessibility/aria/aria_techniques/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: ARIA techniques
+slug: Web/Accessibility/ARIA/ARIA_Techniques
+tags:
+  - ARIA
+  - Accessibility
+  - NeedsTranslation
+  - TopicStub
+translation_of: Web/Accessibility/ARIA/ARIA_Techniques
+---
+### Roles
+
+#### Widget roles
+
+- [Alert](/en/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role "Using the alert role")
```

Comment:
Usually do a find/replace for `\]\(([^)]*) "[^"]*"\)` to `]($1)` to drop the title attributes that have been cleaned out upstream